File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/SalesControlSearchModel.ts

import { BaseSearchModel } from "./baseSearchModel";

export interface SalesControlSearchModel extends BaseSearchModel {
    branchId:number | 0;
    branchCode:string;

    supervisorId:number | 0;
    supervisorCode:string;

    representativeId:number | 0;
    representativeCode:string;

    clientId:number | 0;
    clientCode:string;
    
    fromDate:Date;
    toDate:Date;
}

File: SofiForce.Models/Models/SearchModels/SalesControlSearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class SalesControlSearchModel : BaseSearchModel
    {
        public Int32 BranchId { get; set; }
        public Int32 SupervisorId { get; set; }
        public Int32 RepresentativeId { get; set; }
        public Int32 ClientId { get; set; }


        public DateTime? FromDate { get; set; }
        public DateTime? ToDate { get; set; }


    }
}